assets/css/custom.css

```
@@ -399,14 +399,45 @@ img { height: auto; }
 .repo-card-owner { font-size: .72rem; color: rgb(var(--color-neutral-500)); white-space: nowrap; }
 .repo-card-desc { font-size: .84rem; color: rgb(var(--color-neutral-300)); margin: 0; line-height: 1.4;
   display: -webkit-box; -webkit-line-clamp: 3; -webkit-box-orient: vertical; overflow: hidden; }
 .repo-card-meta { display: flex; flex-wrap: wrap; gap: .5rem; font-size: .72rem; color: rgb(var(--color-neutral-400)); margin-top: auto; }
+/* Topics pills */
+.repo-pills { display: flex; flex-wrap: wrap; gap: .3rem; }
+.repo-pill { font-size: .64rem; padding: .1rem .45rem; border-radius: 999px; background: rgba(var(--color-primary-500), .14);
+  color: rgb(var(--color-primary-300)); border: 1px solid rgba(var(--color-primary-500), .25); }
+/* Badges & tags */
+.repo-badge { font-size: .62rem; padding: .05rem .4rem; border-radius: 999px; background: rgba(var(--color-neutral-600), .4);
+  color: rgb(var(--color-neutral-300)); border: 1px solid rgba(var(--color-neutral-500), .4); }
+.repo-tag { font-size: .62rem; padding: .05rem .4rem; border-radius: 999px; background: rgba(var(--color-warning, .14);
```

Stylelint (17.14.0)

[error] Unclosed bracket (CssSyntaxError)

(CssSyntaxError)